XLS-33d/README.md

```diff
@@ -0,0 +1,594 @@
+
+###### 1.2.1.1.2.3. `Issuer`
+
+The address of the account that controls both the issuance amounts and characteristics of a particular fungible token.
+
+###### 1.2.1.1.2.4. `AssetCode`
+
+A 160-bit blob of data. We recommend using only upper-case ASCII letters, ASCII digits 0 through 9, the dot (`.`) character, and the dash (`-`) character. Dots and dashes should never be the first character of an asset code, and should not be repeated sequentially.
+
+While it's possible to store any arbitrary data in this field, implementations that detect the above recommended character conformance can and should display them as ASCII for human readability, allowing issuers to support well-known ISO-4207 currency codes in addition to custom codes. This also helps prevents spoofing attacks where a [homoglyph](https://en.wikipedia.org/wiki/Homoglyph) might be used to trick a person into using the wrong asset code.
+
+###### 1.2.1.1.2.5. `AssetScale`
+
+An asset scale is the difference, in orders of magnitude, between a standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals 10^(-scale) of a corresponding fractional unit. If the fractional unit equals the standard unit, then the asset scale is 0.
+
+###### 1.2.1.1.2.6. `MaximumAmount`
+
+This value is an unsigned number that specifies the maximum number of CFTs that can be distributed to non-issuing accounts (i.e., `minted`). For issuances that do not have a maximum limit, this value should be set to 0xFFFFFFFFFFFFFFFF.
+
+###### 1.2.1.1.2.7. `OutstandingAmount`
+
+Specifies the sum of all token amounts that have been minted to all token holders. This value can be stored on ledger as a `default` type so that when its value is 0, it takes up less space on ledger. This value is increased whenever an issuer pays CFTs to a non-issuer account, and decreased whenever a non-issuer pays CFTs into the issuing account.
+
+###### 1.2.1.1.2.8. `TransferFee`
+
+This value specifies the fee, in tenths of a [basis point](https://en.wikipedia.org/wiki/Basis_point), charged by the issuer for secondary sales of the token, if such sales are allowed at all. Valid values for this field are between 0 and 50,000 inclusive. A value of 1 is equivalent to 1/10 of a basis point or 0.001%, allowing transfer rates between 0% and 50%. A `TransferFee` of 50,000 corresponds to 50%. The default value for this field is 0.
+
+###### 1.2.1.1.2.9. `OwnerNode`
+
+Identifies the page in the owner's directory where this item is referenced.
+
+##### 1.2.1.1.2. Example **`CFTokenIssuance`** JSON
+
+ ```json
+ {
+     "LedgerEntryType": "CFTokenIssuance",
+     "Flags" : 131072,
+     "Issuer" : "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+     "AssetCode": "5553440000000000000000000000000000000000",
+     "AssetScale": "2",
+     "MaximumAmount", "100000000",
+     "OutstandingAmount", "5",
+     "TransferFee": 50000,     
+     "Metadata" : "",
```

Comment:
Per the above comment, this should be `CFTMetadata` (assuming the JSON property name must align with the SField definition).